test/TEST-01-BASIC/test.sh

```diff
@@ -58,7 +58,7 @@ Type=oneshot
 EOF
 
         setup_testsuite
-    )
+    ) || return 1
```

Comment:
Actually, none of these statements is allowed to fail, right? Did you experiment with `set -e` for these scripts?